hbase-server/src/main/java/org/apache/hadoop/hbase/master/MetaFixer.java

```diff
@@ -77,6 +77,9 @@ void fix() throws IOException {
     }
     fixHoles(report);
     fixOverlaps(report);
+    // Run the ReplicationBarrierCleaner here; it may clear out rep_barrier rows which
+    // can help cleaning up damaged hbase:meta.
+    this.masterServices.runReplicationBarrierCleaner();
```

Comment:
Did we need to handle ReplicationBarrierCleaner.chore running in parallel?  Because HMaster will run ReplicationBarrierCleaner.chore at regular time, and the default interval is 12 hour now...

Reply:
Make the running of the chore exclusive.

Thanks for the review @binlijin